core/services/chainlink/application.go

```diff
@@ -696,7 +699,13 @@ func NewApplication(ctx context.Context, opts ApplicationOpts) (Application, err
 	)
 	srvcs = append(srvcs, bridgeStatusReporter)
 
-	healthChecker := commonservices.NewChecker(static.Version, static.Sha)
+	// healthChecker := commonservices.NewChecker(static.Version, static.Sha)
+	healthChecker := promhealth.NewChecker(static.Version, static.Sha)
+	beholderHealthChecker, err := otelhealth.NewChecker(static.Version, static.Sha, beholder.GetMeter())
```

Comment:
I expanded the API because these constructors cannot be composed. The only change necessary will be:
```suggestion
	healthCfg := commonsrv.HealthCheckerConfig{Ver: static.Version, Sha: static.Sha}
	healthCfg = promhealth.ConfigureHooks(healthCfg)
	healthCfg, err = otelhealth.ConfigureHooks(healthCfg, beholder.GetMeter())
	if err != nil {
		return nil, fmt.Errorf("failed to configure health checker otel hooks: %w", err)
	}
	healthChecker := healthCfg.New()
```

Reply:
And I'm not sure if fetching beholder globally from here is the best option. It would be better to inject it.